src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -3,23 +3,23 @@
 {% trans_default_domain 'search' %}
 
 {{ form_start(form) }}
-<div class="input-group">
+<div class="input-group pl-4">
     {{ form_widget(form.query) }}
 
     <span class="input-group-btn">
-        <button type="submit" class="btn btn-primary">
-            <svg class="ez-icon ez-icon-search-button">
+        <button type="submit" class="btn btn-primary font-weight-bold ez-btn--search">
+            <svg class="ez-icon ez-icon--medium ez-icon--light">
                 <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#search"></use>
             </svg>
-            {{ 'search.perform'|trans|desc('Search') }}</button>
+            <span class="ez-btn--search-label">{{ 'search.perform'|trans|desc('Search') }}</span></button>
```

Comment:
According to BEM methodology, when you are using the modifier `element--smth`, you should also use the element class, like this:
```html
ez-btn ez-btn--search-label
```

Reply:
Understood. Thank you.